Program.cs

```diff
@@ -0,0 +1,167 @@
+            CreateMedicines(ref medicalGoods, new Medicine("Pills", 0, Enums.MedicineTypesEnum.Pills));
+
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Lozenges", 0, Enums.MedicineTypesEnum.Lozenges, "AntiCough"));
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Lozenges", 0, Enums.MedicineTypesEnum.Lozenges, "SoreThroat"));
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Disposables", 0, Enums.MedicineTypesEnum.Disposables, "Gauze"));
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Disposables", 0, Enums.MedicineTypesEnum.Disposables, "BandAid"));
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Liquids", 0, Enums.MedicineTypesEnum.Liquids, "Electrolite"));
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Liquids", 0, Enums.MedicineTypesEnum.Liquids, "Rehyrdration"));
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Pills", 0, Enums.MedicineTypesEnum.Pills, "Antipiretic"));
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Pills", 0, Enums.MedicineTypesEnum.Pills, "APMedicine"));
+            CreateMedicineTypes(ref medicineTypes, new MedicineType("Pills", 0, Enums.MedicineTypesEnum.Pills, "Homeopathy"));
+
+            CreateMedicaments(ref products, new Antipiretic("Nurofen", 102.0, Enums.MedicineTypesEnum.Pills, "Antipiretic", true));
+            CreateMedicaments(ref products, new Antipiretic("Aspirine", 94.0, Enums.MedicineTypesEnum.Pills, "Antipiretic", true));
+            CreateMedicaments(ref products, new Antipiretic("Aspirine", 94.0, Enums.MedicineTypesEnum.Pills, "Antipiretic", true));
+            CreateMedicaments(ref products, new Antipiretic("Paracetamol", 29.0, Enums.MedicineTypesEnum.Pills, "Antipiretic", true));
+            CreateMedicaments(ref products, new APmedicine("Enalapril", 33.0, Enums.MedicineTypesEnum.Pills, "APMedicine", true));
+            CreateMedicaments(ref products, new APmedicine("Labetalol", 21.0, Enums.MedicineTypesEnum.Pills, "APMedicine", true));
+            CreateMedicaments(ref products, new APmedicine("Benzohexonium", 11.0, Enums.MedicineTypesEnum.Pills, "APMedicine", false));
+            CreateMedicaments(ref products, new APmedicine("Nifedipine", 23.0, Enums.MedicineTypesEnum.Pills, "APMedicine", true));
+            CreateMedicaments(ref products, new Homeopathy("VertigoHeel", 194.0, Enums.MedicineTypesEnum.Pills, "Antipiretic", true));
+            CreateMedicaments(ref products, new Homeopathy("CrapHeel", 944.7, Enums.MedicineTypesEnum.Pills, "Antipiretic", true));
+            CreateMedicaments(ref products, new Homeopathy("SnakeOil", 994.0, Enums.MedicineTypesEnum.Pills, "Antipiretic", true));
+        }
+
+        public static void PharmacyInterface(Medicine[] products)
+        {
+            int optionCounter = 3;
```

Comment:
Assume the whole console interaction flow should be extracted into the separate Console/Output Service + separate methods there would be needed to keep the code clean and shiny